src/instance.jl

```diff
@@ -326,15 +337,15 @@ Let `MOI` denote `MathOptInterface`, `MOIU` denote `MathOptInterfaceUtilities` a
 The macro would create the types:
 ```julia
 struct LPInstanceScalarConstraints{T, F <: MOI.AbstractScalarFunction} <: MOIU.Constraints{F}
-    equalto::Vector{MOIU.C{F, MOI.EqualTo{T}}}
-    greaterthan::Vector{MOIU.C{F, MOI.GreaterThan{T}}}
-    lessthan::Vector{MOIU.C{F, MOI.LessThan{T}}}
-    interval::Vector{MOIU.C{F, MOI.Interval{T}}}
+    EqualTo::Vector{MOIU.C{F, MOI.EqualTo{T}}}
```

Comment:
why this renaming?

Reply:
This is commented above (do Ctrl+F "lowercase"). Julia v0.7 moved lowercase to Unicode. To avoid complicating the code just for the name of fields no one will look at, I prefered removing this lowercase (at least for now).

Reply:
Hmm. Why not use a dictionary to map from symbols to types? Breaking the convention for field names is confusing.

Reply:
I have added `using Unicode` with a `if` for the correct version.